pkg/pillar/hypervisor/kvm.go

```diff
@@ -800,6 +816,20 @@ func (ctx KvmContext) CreateDomConfig(domainName string, config types.DomainConf
 				logrus.Infof("Adding serial <%s>\n", ib.Serial)
 				serialAssignments = addNoDuplicate(serialAssignments, ib.Serial)
 			}
+			if ib.Type == types.IoLCAN && ib.Ifname != "" {
+				var canIfName string
```

Comment:
Below you call the same concept canHostIf. I think that name is more clear and it makes sense to use the same variable naming approach.